samples/Calling/src/app/utils/AppUtils.ts

```diff
@@ -37,6 +41,43 @@ export const getGroupIdFromUrl = (): GroupLocator | undefined => {
 
 export const createGroupId = (): GroupLocator => ({ groupId: generateGUID() });
 
+/* @conditional-compile-remove(rooms) */
+export const createRoomId = async (): Promise<string> => {
+  const requestOptions = {
+    method: 'POST',
+    headers: {
+      Host: 'alkwa-fn-test.azurewebsites.net',
+      'Content-Type': 'application/json',
+      'Referrer-Policy': 'no-referrer'
+    }
+  };
+  const response = await fetch('http://localhost:7071/api/Rooms-CreateRoom', requestOptions);
+  if (!response.ok) {
+    throw 'Invalid token response';
+  }
+
+  const body = await response.json();
+  return body['id'];
+};
+
+/**
+ * Joins an ACS room with a given roomId and role
+ */
+/* @conditional-compile-remove(rooms) */
+export const joinRoom = async (userId: string, roomId: string, role: Role): Promise<void> => {
+  const requestOptions = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({ acsUserId: userId, roomId: roomId, role: role })
+  };
+  const response = await fetch('http://localhost:7071/api/Rooms-AddParticipants', requestOptions);
```

Comment:
can we export this localhost:7071 url to a common variable? I think in the future we are going to ask all backend functionality coming from a target url?